syz-manager/repro.go

```diff
@@ -67,11 +67,15 @@ func newReproManager(mgr reproManagerView, reproVMs int, onlyOnce bool) *reproMa
 // startReproduction() is assumed to be called only once.
 // The agument is the maximum number of VMs dedicated to the bug reproduction.
 func (m *reproManager) StartReproduction() {
-	log.Logf(1, "starting reproductions (max %d VMs)", m.reproVMs)
-
-	for count := 1; m.calculateReproVMs(count) <= m.reproVMs; count++ {
+	count := 0
+	for {
```

Comment:
More canonical loop form:
`for ; m.calculateReproVMs(count+1) <= m.reproVMs; count++ {`